src/Integration/Exclusions/ExclusionSettingsStorage.cs

```diff
@@ -66,7 +66,7 @@ public void SaveSettings(ServerExclusions settings)
                 throw new InvalidOperationException("Cannot save exclusions in Standalone mode.");
             }
 
-            var fileContent = JsonConvert.SerializeObject(settings);
+            var fileContent = JsonConvert.SerializeObject(settings, Formatting.Indented);
```

Comment:
Minor tweak - I was looking at how Roslyn suppressions are handled, and found the un-indented json a pain to work with. Most of the time we indent it. This PR changes the two places we write un-indented code.